src/js/actions/transform.js

```diff
@@ -1044,10 +1050,18 @@ define(function (require, exports) {
             }
         }, this);
 
+        _moveToArtboardHandler = function () {
+            var appStore = this.flux.store("application"),
```

Comment:
Since you're in here touching this, it would probably be better to make this an action that transfers to `resetIndex`.